packages/flutter/lib/fix_data/fix_material/fix_selectable_text.yaml

```diff
@@ -0,0 +1,32 @@
+# Copyright 2014 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+# For details regarding the *Flutter Fix* feature, see
+# https://flutter.dev/docs/development/tools/flutter-fix
+
+# Please add new fixes to the top of the file, separated by one blank line
+# from other fixes. In a comment, include a link to the PR where the change
+# requiring the fix was made.
+
+# Every fix must be tested. See the flutter/packages/flutter/test_fixes/README.md
+# file for instructions on testing these data driven fixes.
+
+# For documentation about this file format, see
+# https://dart.dev/go/data-driven-fixes.
+
+# * Fixes in this file are for SelectableText from the material library. *
+version: 1
+transforms:
+  # Change made in https://github.com/flutter/flutter/pull/128522
+  - title: "Migrate to 'textScaler'"
+    date: 2023-06-09
+    element:
+      uris: [ 'material.dart' ]
+      getter: 'textScaleFactor'
+      inClass: 'SelectableText'
+    changes:
+      - kind: 'rename'
+        newName: 'textScaler?.textScaleFactor'
```

Comment:
I think we should remove all the dartfixes that migrate you to another deprecated API. What is the point of that? To me, it just creates more churn and is not saving you any work. In fact, it may actually make the real migration to textscaler harder because now the starting point is not some known code you wrote, but some unkown code we migrated you to.

Reply:
The dart fixes that migrate you to the real TextScaler API (e.g. that replace `textScaleFactor: 2` with `textScaler: TextScaler.linear(2)` are great, though! Those are actually saving people work and make their lives easier.

Reply:
got rid of these fixes since nobody likes em